src/py/flwr/server/run_serverapp.py

```diff
@@ -136,32 +137,50 @@ def run_server_app() -> None:
             cert_path,
         )
 
-    log(
-        DEBUG,
-        "Flower will load ServerApp `%s`",
-        getattr(args, "server-app"),
+    if not (getattr(args, "server-app") is None) ^ (args.run_id is None):
```

Comment:
We should call this only once and then use the read value below:
```suggestion
    server_app_ref: str = getattr(args, "server-app")
    if not (server_app_ref is None) ^ (args.run_id is None):
```